### PR TITLE
chore: SCIM settings UI

### PR DESCRIPTION
**Pull request by nunogois**
https://linear.app/unleash/issue/2-1981/ui-allow-users-to-set-up-scim

Adds UI for Provisioning (SCIM) settings.

image

image

image

### CODE REVIEW

File: frontend/src/component/admin/auth/ScimSettings/ScimSettings.tsx
Comment (sighphyre) [inline frontend/src/component/admin/auth/ScimSettings/ScimSettings.tsx:70]: Thanks! Now I know where to put the docs xD

**Review by sighphyre — APPROVED** [review]
LGTM! But I'd wait for someone with good Ueyes before hitting merge

**Review by FredrikOseberg — COMMENTED** [review]
I think this is fine for now. The entire SSO section needs to be revamped at some point but that is another story and beyond the scope of this. I feel like this keeps the theme of what we have so far pretty well.

**Pull request merged**
